apps/prairielearn/package.json

```diff
@@ -215,7 +215,6 @@
     "mocha": "^10.2.0",
     "mocha-steps": "^1.3.0",
     "request": "^2.88.2",
-    "sinon": "^15.2.0",
```

Comment:
This will require a conflict resolution before merging, since it has been upgraded already.

Reply:
Resolved in fc309c0.